prowler/providers/aws/services/ec2/ec2_instance_account_imdsv2_enabled/ec2_instance_account_imdsv2_enabled.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "ec2_instance_account_imdsv2_enabled",
+  "CheckTitle": "EC2 instances should use Instance Metadata Service Version 2 (IMDSv2) to protect against SSRF vulnerabilities, you can set the default instance metadata version at the account level for each AWS Region",
+  "CheckType": [
+    "Data Protection"
+  ],
+  "ServiceName": "ec2",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id",
+  "Severity": "medium",
+  "ResourceType": "AwsEc2Instance",
+  "Description": "Ensure that EC2 instances use Instance Metadata Service Version 2 (IMDSv2) to protect against SSRF vulnerabilities.",
+  "Risk": "EC2 instances that use IMDSv1 are vulnerable to SSRF attacks.",
+  "RelatedUrl": "https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-IMDS-new-instances.html#set-imdsv2-account-defaults",
+  "Remediation": {
+    "Code": {
+      "CLI": "aws ec2 modify-instance-metadata-defaults --region <region> --http-tokens required --http-put-response-hop-limit 2",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": ""
+    },
+    "Recommendation": {
+      "Text": "Enable Instance Metadata Service Version 2 (IMDSv2) on the EC2 instances. For more information, see the AWS documentation.",
```

Comment:
Mention that it is in the account level, please.

Reply:
Done